### ISSUE TITLE
neet password to resetore hosts file

### ISSUE THREAD

**Issue by nailuoGG** [2ndalpha/gasmask#84]
![image](https://cloud.githubusercontent.com/assets/4554827/11759716/757e833a-a0bc-11e5-915a-6932dad630ca.png)

my gas-mask version: 0.8.3
my system: OS X 10.11.2 (15C50)

**Comment by hao-lh** [comment]
Yes, it also occurred frequently after I have updated to new newer version of OS X 10.11.2.

**Comment by bennyp123** [comment]
I've got this as well, I think it's a result of Covenant Eyes repeatedly overwriting /etc/hosts. If there's anything you can do to solve this.
